sdk/cognitiveservices/cognitiveservices-qnamaker/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure/cognitiveservices-qnamaker",
   "author": "Microsoft Corporation",
   "description": "QnAMakerClient Library with typescript type definitions for node.js and browser.",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
should this be 4.0.0?

Reply:
I had this initially to 4.0.0, however @ramya-rao-a suggested to make it a minor bump since there were no breaking changes

Reply:
That's fine, I was just confused by the title of the PR :)